cli/tests/program.rs

```diff
@@ -488,13 +488,27 @@ fn test_cli_program_deploy_with_authority() {
         program_data[..]
     );
 
-    // Set a new authority
+    // Set a new authority sign offline first
     let new_upgrade_authority = Keypair::new();
     config.signers = vec![&keypair, &upgrade_authority];
     config.command = CliCommand::Program(ProgramCliCommand::SetUpgradeAuthority {
         program_pubkey,
         upgrade_authority_index: Some(1),
         new_upgrade_authority: Some(new_upgrade_authority.pubkey()),
+        sign_only: true,
+        dump_transaction_message: false,
```

Comment:
shouldn't this also take the blockhash query?
```suggestion
        sign_only: true,
        dump_transaction_message: false,
        blockhash_query: BlockhashQuery::new(Some(blockhash), true, None),
```